breathecode/authenticate/views.py

```diff
@@ -271,6 +271,28 @@ def post(self, request, academy_id=None):
     @capable_of('crud_member')
     def put(self, request, academy_id=None, user_id_or_email=None):
 
+        mentor = MentorProfile.objects.filter(id=mentor_id, services__academy__id=academy_id).first()
```

Comment:
A staff member can not have MentorProfile, remove this validation, this is not needed